backends/python-fastapi/tests/conftest.py

```
@@ -136,18 +148,7 @@ def make_staff(db, **kwargs) -> StaffModel:
-    private_key = rsa.generate_private_key(public_exponent=65537, key_size=2048)
-    pub_pem = private_key.public_key().public_bytes(
-        serialization.Encoding.PEM,
-        serialization.PublicFormat.SubjectPublicKeyInfo,
-    ).decode()
-    priv_pem = private_key.private_bytes(
-        serialization.Encoding.PEM,
-        serialization.PrivateFormat.TraditionalOpenSSL,
-        serialization.NoEncryption(),
-    ).decode()
-
+    """RSA キーペア付きのクライアントレコードを作成します（キャッシュ済みキーを再利用）。"""
```

Ruff (0.15.14)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)